src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -66,6 +68,62 @@ public static String formatEvent(String header, Event event) {
         return formattedString;
     }
 
+    /**
+     * Formats a provided event to an ArrayList format.
+     *
+     * @param event Event to be formatted
+     * @return ArrayList of Strings to represent the Event.
+     */
+    public static ArrayList<String> formatEvent(Event event) {
+        ArrayList<String> result = new ArrayList<>();
+        result.add("Event: " + event.getTitle());
+        result.add("Date: " + event.getDate().toString() + "\tTime: " + event.getTime().toString());
+        result.add("Reminder: " + event.getToRemind());
+        String repeatingString = "Repeating: " + event.getRecurring();
+        String endRecurrenceDateString = "";
+        if (event instanceof RecurringEvent) {
+            RecurringEvent recurringEvent = (RecurringEvent) event;
+            endRecurrenceDateString = recurringEvent.getEndRecurrenceString();
+        }
+        result.add(repeatingString + endRecurrenceDateString);
+        return result;
```

Comment:
Do the enclosing here and just make it return the formatted string straightaway. Can overload it to take in a header for printing 1 event.

Reply:
What do you mean?

Reply:
Oh I did it as a separate function for compatability to other PR where I do listing. No point re-writing code just to iterate the same lines.

Reply:
My considerations are formatEvent would be heavily utilized when listing events. When listing multiple events we would most likely be listing them in the same box so it would make sense to have a separate function that formats them as a string.

Reply:
Noted on this.